### DISCUSSION TITLE
[General] <Native bridge L2 to L1>

### DISCUSSION THREAD

**Discussion by xinparkn** [zkSync-Community-Hub/zksync-developers#285] [General]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

I want to comfirm  a problem.

### What is the specific issue or error you're encountering?

Native bridge L2 to L1 for Withdraw.
after 26hours , still is not  come in my L1 account.

### Can you share the error messages or logs you're receiving, if any?

No error messages.
Transaction Hash: 0xb48f96fb73cb1c4985e5065e6b56033eab71484b529e54be676a3b2daf474757
Status: Success
Block: 2067 L1 Block Confirmations
Timestamp:1 day 4 hrs ago (Jan-22-2024 01:15:49 AM +UTC)

### Have you made any recent changes to the contract before encountering this issue?

no

### Are there any external libraries or contracts that your contract interacts with?

no

### Can you provide the relevant portions of your contract code where the issue is occurring?

i dont know

### Have you tried to isolate the problem, and if so, what were the results?

i dont know

### What steps have you already taken to try to resolve the issue?

no

**Comment by xinparkn** [comment]
i kown what to do . so closed

**Comment by MexicanAce** [comment]
Closing discussion as self-resolved
